k9mail/src/main/java/com/fsck/k9/view/MessageWebView.java

```diff
@@ -113,8 +115,11 @@ private void disableDisplayZoomControls() {
      * @param text
      *      The message body to display.  Assumed to be MIME type text/html.
      */
-    public void setText(String text) {
-     // Include a meta tag so the WebView will not use a fixed viewport width of 980 px
+    public void setText(String text, AttachmentResourceProvider attachmentResourceProvider) {
```

Comment:
This method now does more than one thing. Also, the name doesn't really fit anymore.

Suggestion: Create a method for each basic operation and then a public method that calls both of these methods.